### PR TITLE
chore: add standardization and configuration for markdownlint

### PR DESCRIPTION
**Pull request by pbredenberg**

### CODE REVIEW

File: coding-standards.md [hunk 1]
```diff
@@ -279,7 +280,11 @@ class SomeClass {
       * But, but, but, why do I need newlines at the end of my files? See [this
         answer](http://stackoverflow.com/a/729795/1011988) on StackExchange. Then try
         this:
-         * `echo -n 'this is a test' > /tmp/test-nonewline.txt && echo 'this is a test' > /tmp/test-newline.txt && wc -l /tmp/test-*newline.txt`
+
+         ```bash
+          echo -n 'this is a test' > /tmp/test-nonewline.txt && echo 'this is a test' > /tmp/test-newline.txt && wc -l /tmp/test-*newline.txt`
+        ```
```
Comment (onebytegone) [inline coding-standards.md:286]:
```suggestion
         ```
```

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: coding-standards.md (hunk 1 above)
Comment (pbredenberg) [reply coding-standards.md:286]: You'd like it if these columns line dup, wouldn't you?

Comment (pbredenberg) [reply coding-standards.md]: Fixed

File: coding-standards.md [hunk 2]
```diff
@@ -280,7 +280,11 @@ class SomeClass {
       * But, but, but, why do I need newlines at the end of my files? See [this
         answer](http://stackoverflow.com/a/729795/1011988) on StackExchange. Then try
         this:
-         * `echo -n 'this is a test' > /tmp/test-nonewline.txt && echo 'this is a test' > /tmp/test-newline.txt && wc -l /tmp/test-*newline.txt`
+
+         ```bash
+          echo -n 'this is a test' > /tmp/test-nonewline.txt && echo 'this is a test' > /tmp/test-newline.txt && wc -l /tmp/test-*newline.txt`
```
Comment (onebytegone) [inline coding-standards.md:285]: Sorry, just noticed the extra space before `echo`

```suggestion
         echo -n 'this is a test' > /tmp/test-nonewline.txt && echo 'this is a test' > /tmp/test-newline.txt && wc -l /tmp/test-*newline.txt`
```

**Review by onebytegone — CHANGES_REQUESTED** [review]

File: coding-standards.md (hunk 2 above)
Comment (pbredenberg) [reply coding-standards.md]: That whole fence was all messed up. Should be good now.

**Review by onebytegone — APPROVED** [review]

**Pull request merged**